<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QSBLT4Q" ignored="0" name="Default test" tests="4" failures="0" timestamp="19 Sep 2017 12:32:31 GMT" time="0.053" errors="0">
  <testcase name="@BeforeTest launchapp" time="0.01" classname="tests.TestLogin">
    <failure type="java.io.FileNotFoundException" message="test-configurations.txt (The system cannot find the file specified)">
      <![CDATA[java.io.FileNotFoundException: test-configurations.txt (The system cannot find the file specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.util.Scanner.<init>(Scanner.java:611)
	at tests.Tester.getDriverPath(Tester.java:17)
	at tests.TestLogin.launchapp(TestLogin.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:658)
	at org.testng.TestRunner.run(TestRunner.java:626)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest launchapp -->
  <testcase name="@AfterTest terminatetest" time="0.0" classname="tests.TestLogin">
    <skipped/>
  </testcase> <!-- @AfterTest terminatetest -->
  <testcase name="testEmptyUsername" time="0.0" classname="tests.TestLogin">
    <skipped/>
  </testcase> <!-- testEmptyUsername -->
  <testcase name="testEmptyPassword" time="0.0" classname="tests.TestLogin">
    <skipped/>
  </testcase> <!-- testEmptyPassword -->
  <testcase name="testWrongCredentials" time="0.0" classname="tests.TestLogin">
    <skipped/>
  </testcase> <!-- testWrongCredentials -->
  <testcase name="testLogin" time="0.0" classname="tests.TestLogin">
    <skipped/>
  </testcase> <!-- testLogin -->
</testsuite> <!-- Default test -->
